common/app/common/pages/ImageData.java

```diff
@@ -0,0 +1,36 @@
+package common.pages;
+
+import io.sphere.sdk.models.Base;
+import io.sphere.sdk.models.Image;
+
+public class ImageData extends Base {
+
+    private final Image image;
+
+    private ImageData(final Image image) {
+        this.image = image;
+    }
+
+    public static ImageData of(final Image image) {
+        return new ImageData(image);
+    }
+
+    public String getThumbImage() {
+        return nullIfEmpty(image.getUrl());
+    }
+
+    public String getBigImage() {
+        return nullIfEmpty(image.getUrl());
```

Comment:
We have to take a look into it, because in Handlebars.js it says:
```
You can use the if helper to conditionally render a block. If its argument returns false, undefined, null, "", 0, or [], Handlebars will not render the block.
```
http://handlebarsjs.com/builtin_helpers.html

I'll check in Handlebars.java, maybe it's a change of behaviour.